### PR TITLE
webpack-feature

### PR DESCRIPTION
**Pull request by abiy006**
Tasks accomplished on this PR:

- webpack is configured
- linter is configured
- todo list is added

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @abiy006 , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a project with a blank readme. please use this [template](https://github.com/microverseinc/readme-template)
inv


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -20,14 +20,14 @@ const arrayToDo = [
...
-  h1.style.borderBottom = '1px solid gray';
-  mytodoo.appendChild(h1);
+  // const h1 = document.getElementById('h1id');
+  // h1.textContent = "Today's To Do";
+  // h1.style.display = 'flex';
+  // h1.style.height = '3rem';
+  // h1.style.fontSize = '3rem';
+  // h1.style.alignItems = 'center';
+  // h1.style.borderBottom = '1px solid gray';
+  // mytodoo.appendChild(h1);
```
Comment (Abenezer-Tilahun) [inline src/index.js:30]: - [ Optional ] Kindly I recommend you to remove the commented code to make your code base clean and not ambiguous.

File: README.md [hunk 2]
```diff
@@ -1 +1,160 @@
...
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [WEBPACK PRJECT] <a name="about-project"></a>
+
+**[Abiy Capstone One]** This is a webpack project created during my Microverse course.
```
Comment (Abenezer-Tilahun) [inline README.md:40]: - [Optional ] Kindly I Recommend you to change  **[Abiy Capstone One]** to the current project activity.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @abiy006 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented To Do list: list structure.

## Optional suggestions 
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**